torch/nn/quantized/modules/pooling.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```

Comment:
can we do this in quantized/modules/__init__.py?

Reply:
I am still fighting the Python community about that. The current hack is to use it, but to be careful with `from ... import *`

Reply:
oh, I mean do we need this file? we can just create a alias in `quantized/modules/__init__.py`

Reply:
This is to mirror non-quantized version. Technically we don't need it, but creating an alias will prevent from `from torch.nn.quantized.pooling import MaxPool2d`

Reply:
yeah I mean we can put this in init? and i think people will do `from torch.nn.quantized import MaxPool2d`

Reply:
Yes. I think right now the users can still do `from torch.nn.quantized import MaxPool2d`. Do you think I should move the alias to init, and not create a new file for it?

Reply:
removed the pooling -- now only __init__ has it